config/config.go

```diff
@@ -257,20 +252,20 @@ func (c *Config) GetCurrencyConfig() CurrencyConfig {
 
 // GetExchangeBankAccounts returns banking details associated with an exchange
 // for depositing funds
-func (c *Config) GetExchangeBankAccounts(exchangeName string, depositingCurrency string) (BankAccount, error) {
+func (c *Config) GetExchangeBankAccounts(exchangeName, depositingCurrency string) (BankAccount, error) {
 	m.Lock()
 	defer m.Unlock()
 
-	for _, exch := range c.Exchanges {
-		if exch.Name == exchangeName {
-			for _, account := range exch.BankAccounts {
+	for x := range c.Exchanges {
```

Comment:
Is iterating on the index vs putting it in a variable here better practice, more efficient or both?

Reply:
An index or pointer is the most efficient, doing it the original way allocates an exchange config object per iteration.